packages/core/src/llm/LLM.ts

```diff
@@ -613,10 +608,28 @@ If a question does not make any sense, or is not factually coherent, explain why
   }
 }
 
+export const ALL_AVAILABLE_ANTHROPIC_V2_MODELS = {
+  "claude-2.1": {
+    contextWindow: 200000,
+  },
+  "claude-instant-1.2": {
+    contextWindow: 100000,
+  },
+};
+
+export const ALL_AVAILABLE_V3_MODELS = {
+  "claude-3-opus": { contextWindow: 200000 },
+  "claude-3-sonnet": { contextWindow: 200000 },
+};
+
 export const ALL_AVAILABLE_ANTHROPIC_MODELS = {
-  // both models have 100k context window, see https://docs.anthropic.com/claude/reference/selecting-a-model
-  "claude-2": { contextWindow: 200000 },
-  "claude-instant-1": { contextWindow: 100000 },
+  ...ALL_AVAILABLE_ANTHROPIC_V2_MODELS,
+  ...ALL_AVAILABLE_V3_MODELS,
+};
+
+const AVAILABLE_ANTHROPIC_MODELS_WITHOUT_DATE: { [key: string]: string } = {
+  "claude-3-opus": "claude-3-opus-20240229",
```

Comment:
`claude-3-opus-20240229` should be the correct model name, I would like to as-is

https://docs.anthropic.com/claude/reference/messages-streaming#streaming-with-sdks

Reply:
@himself65 I asked Emanuel to add this feature. We're seeing other people do the same.